lib/Dialect/Triton/IR/Ops.cpp

```diff
@@ -503,6 +503,8 @@ llvm::SmallVector<Type> ReduceOp::getElementTypes() {
   return getElementTypesImpl(this->getOperands());
 }
 
+// Returns the CombineOp iff this ReduceOp's region contains only
```

Comment:
This should go to the header file--please embed it in the op defintion's `extraClassDeclaration`.

Reply:
Got it, didn't realize we can put comments there.